Update README.md (#26)
exasol/parquet-edml-generator: README.md

## Installation

> Download the binary for your OS from the latest [Releases](https://github.com/exasol/parquet-edml-generator/releases).

Reviewer: ```suggestion
Download the binary for your operating system from the latest [release](https://github.com/exasol/parquet-edml-generator/releases).
```

## Usage

### Windows

Run on your local PC:

```shell
parquet-edml-generator.exe <parquet file>
```
### Linux

Run on your local PC:

```shell

> ./parquet-edml-generator-linux-binary  <parquet file>

Reviewer: ```suggestion
./parquet-edml-generator-linux-binary <parquet file>
```

```

### Mac OS

Run on your local PC:

```shell

> ./parquet-edml-generator-mac-binary  <parquet file>

Reviewer: ```suggestion
./parquet-edml-generator-mac-binary <parquet file>
```
